platforms/pf-robot-motion-control/motion_control.cpp

```diff
@@ -477,6 +476,26 @@ void pf_motor_drive(const cogip::cogip_defs::Polar &command)
     int16_t left_command = (int16_t) std::max(std::min(command.distance() - command.angle(), (double)(std::numeric_limits<int16_t>::max()) / 2),
                                                   (double)(std::numeric_limits<int16_t>::min()) / 2);
 
+    if (pf_motion_control_platform_engine.pose_reached() == cogip::motion_control::target_pose_status_t::blocked) {
```

Comment:
Not mandatory for today in my mind but it can be good to add a method named `is_blocked` or renaming this method with something like `get_pos_status`. Its name will perhaps be more meaningful for this specific case.

Reply:
Yes due to legacy, it was just a boolean in the past. I think we will rework this in coming weeks with moving platform to c++.